src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,38 +1,51 @@
 #!/usr/bin/python
 import sys
 import swsssdk
+import argparse
 
-argc = len(sys.argv)
-if argc == 2 and sys.argv[1] == '-h':
-    print("""
-Example 1: sonic-db-cli CONFIG_DB keys *
-Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
-Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
-Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
-""")
-elif argc < 3:
-    msg = "'Usage: sonic-db-cli <db_name> <cmd> [arg [arg ...]]'. See 'sonic-db-cli -h' for detail examples."
-    print >> sys.stderr, msg
-else:
-    dbname = sys.argv[1]
-    dbconn = swsssdk.SonicV2Connector(use_unix_socket_path=False)
-    try:
-        dbconn.connect(dbname)
-    except RuntimeError:
-        msg = "Invalid database name input : '{}'".format(sys.argv[1])
-        print >> sys.stderr, msg
-    else:
-        client = dbconn.get_redis_client(dbname)
-        args = sys.argv[2:]
-        resp = client.execute_command(*args)
-        """
-        sonic-db-cli output format mimic the non-tty mode output format from redis-cli
-        based on our usage in SONiC, None and list type output from python API needs to be modified
-        with these changes, it is enough for us to mimic redis-cli in SONiC so far since no application uses tty mode redis-cli output
-        """
-        if resp is None:
-            print ""
-        elif isinstance(resp, list):
-            print "\n".join(resp)
+'''
+  Examples of how to use this script 
+      Example 1: sonic-db-cli -ns asic0 CONFIG_DB keys *
+      Example 2: sonic-db-cli -ns asic1 APPL_DB HGETALL VLAN_TABLE:Vlan10
+      Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
+      Example 4: sonic-db-cli -ns asic5 APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
+'''
+
+def main():
+    parser = argparse.ArgumentParser(description='SONiC DB CLI',
+                         version='1.0.0',
+                         formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('-ns', '--namespace', type=str, help='Namespace to use [asic0,asic1...,asicn]', default='default')
+    parser.add_argument('database', nargs='?', type=str, help='Database name')
```

Comment:
>database [](start = 25, length = 8)

add `required`, so no need to check yourself.

Reply:
Since these are positional arguments, cannot add the required option.